lua/neogit/popups/commit.lua

```diff
@@ -163,7 +163,7 @@ local function create()
             -- we need \r? to support windows
             data = split(data, '\r?\n')
             await(prompt_commit_message(data, skip_gen))
-            local _, code = await(cli.commit.commit_message_file(COMMIT_FILE).args(unpack(popup.get_arguments())).call())
+            local _, code = await(cli.commit.commit_message_file(COMMIT_FILE).args(unpack(popup.get_arguments())).call()) --luacheck: ignore
```

Comment:
why are you ignoring this line?

Reply:
Topo long. Line breaks look a bit look a bit weird but they are probably the better alternative